src/redis.cr

```diff
@@ -260,7 +286,7 @@ class Redis
   # :nodoc:
   def command(request : Request)
     with_reconnect do
-      client.strategy.command(request).as(RedisValue)
+      strategy.command(request).as(RedisValue)
```

Comment:
you not garanted that strategy would not be nil here, this is just suggestion, what if it would be.
When crystal introduce multithreading, this is became bug.

Reply:
If `reconnect` is `false`, then there would be no strategy - true. This could be handled more gracefully by telling the caller that he is issueing a command on a disconnected `Redis` instance.

Though this has nothing to do with multithreading, since a `Redis` instance must always be "owned" by a single thread/fiber.